variables.md

```diff
@@ -96,6 +94,8 @@ LIVECD | boolean | false | Indicates live image being used.
 LIVE_INSTALLATION | boolean | false | If set, boots the live media and starts the builtin NET installer.
 LIVE_UPGRADE | boolean | false | If set, boots the live media and starts the builtin NET installer in upgrade mode.
 LIVETEST | boolean | false | Indicates test of live system.
+LTP_COMMAND_FILE | string | | The LTP test command file (e.g. syscalls, cve)
+LTP_COMMAND_EXCLUDE | string | | This regex is used to exclude tests from LTP command file.
```

Comment:
LGTM. FYI we document these variables already in tests/kernel/run_ltp.pm. Also there are more variables, LTP_COMMAND_PATTERN in "our" LTP scripts. Not sure if it's a good idea to document some variables here and some in test modules.

Reply:
I would favor for `variables.md`, which is more visible.